lib/membrane_live_web/channels/event_channel.ex

```diff
@@ -52,12 +54,13 @@ defmodule MembraneLiveWeb.EventChannel do
         with {:ok, %{"user_id" => uuid}} <- Tokens.auth_decode(token),
              {:ok, name} <- Accounts.get_username(uuid),
              {:ok, email} <- Accounts.get_email(uuid),
+             is_moderator <- Webinars.check_is_user_moderator(uuid, id),
+             socket <- Socket.assign(socket, %{is_moderator: is_moderator, event_id: id}),
              [] <- Presence.get_by_key(socket, email),
-             {:ok, socket} <- create_event_stream(id, socket),
-             {:ok, is_presenter} = check_if_presenter(email, reloaded, id),
-             {:ok, is_request_presenting} = check_if_request_presenting(email, reloaded, id),
-             is_moderator <- Webinars.check_is_user_moderator(uuid, id) do
-          {:ok, socket} = if is_presenter, do: join_event_stream(socket), else: {:ok, socket}
+             {:ok, socket} <- create_event(socket, is_moderator),
```

Comment:
As now `is_moderator` is inside the socket the arrity of `create_event` can be changed to 1.

Reply:
Fixed.